astro_2.0/components/DaosList/DaosList.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, ReactNode, useCallback } from 'react';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
+
+import { Button } from 'components/button/Button';
+import { CREATE_DAO_URL } from 'constants/routing';
+
+import { useAuthContext } from 'context/AuthContext';
+
+import styles from './DaosList.module.scss';
+
+interface DaosListProps {
+  label: string;
+  children: ReactNode;
```

Comment:
You shouldn't add `children` in props as FC will provide this prop by default and in a bit more extended way.